### ISSUE TITLE
Add documentation for how to call this API

### ISSUE THREAD

**Issue by azohra** [wearefriday/spectre_client#2]
Hey folks,

I dug into the gem to solve this but I think an example should be on the Readme.

```
    spectre.submit_test(source_url: @current_page.current_url, name: key, browser: BROWSER, size: 1200, screenshot: ChunkyPNG::Image.from_file(dest.path) )
```

would be great to publicize what the available optional keys are that can be set and what values they expect. It took me a while to determine size wanted an int and that screenshot wanted an actual image string.

**Comment by nickdunn** [comment]
Good point, docs are lacking!

They should all be strings as far as I can remember, including size. The reason being that you might want to log size as "mobile" or "portrait" or "1200px" or whatever makes sense to categorise your tests.

I'll make a todo to add some better API documentation.

**Comment by nickdunn** [comment]
Addressed in #4.
